tests/suites/test_suite_pk.function

```diff
@@ -90,28 +90,64 @@ size_t mbedtls_rsa_key_len_func( void *ctx )
     return( ((const mbedtls_rsa_context *) ctx)->len );
 }
 #endif /* MBEDTLS_RSA_C */
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+
+#include "mbedtls/psa_util.h"
+
+#define PK_PSA_INVALID_SLOT 0 /* guaranteed invalid */
```

Comment:
How about adding such define in the PSA utils? Just a thought, not intended for this PR. It's used in several places and different PR's though.